staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -185,6 +187,7 @@ func (o *DebugOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.Image, "image", o.Image, i18n.T("Container image to use for debug container."))
 	cmd.Flags().StringToStringVar(&o.SetImages, "set-image", o.SetImages, i18n.T("When used with '--copy-to', a list of name=image pairs for changing container images, similar to how 'kubectl set image' works."))
 	cmd.Flags().String("image-pull-policy", "", i18n.T("The image pull policy for the container. If left empty, this value will not be specified by the client and defaulted by the server."))
+	cmd.Flags().StringSlice("image-pull-secrets", nil, i18n.T("The image pull secrets to use for pulling images from private container image registries."))
```

Comment:
Just wondering, if we run only a single container using `kubectl debug`, why would one need to specify **multiple** image pull secrets?

Reply:
Ah, you are right. We don't need multiple image pull secrets. I adjusted the code only to receive one image pull secret.